fleetconfig-controller/api/v1alpha1/fleetconfig_types.go

```diff
@@ -29,6 +29,7 @@ type FleetConfigSpec struct {
 	Hub              Hub               `json:"hub"`
 	Spokes           []Spoke           `json:"spokes"`
 	RegistrationAuth *RegistrationAuth `json:"registrationAuth,omitempty"`
+	AddOns           []*AddOnConfig    `json:"addOns,omitempty"`
```

Comment:
Id suggest renaming this to AddOnConfigs so that there isn't any confusion between the spoke specific addOns and these addOns.


```suggestion
	AddOnConfigs           []*AddOnConfig    `json:"addOnConfigs,omitempty"`
```